fiftyone/server/routes/embeddings.py

```diff
@@ -25,7 +25,6 @@
     fof.BooleanField,
     fof.IntField,
     fof.FloatField,
```

Comment:
we don't support customize color by FloatField through ColorScheme in the app yet. Do we want to be consistent here?

Reply:
Coloring by float is useful (and fully supported) by the embeddings panel. It will show a continuous colorbar.

I believe the colorscale is pulled from dataset.app_config.colorscale, which is also used by Heatmap fields